.github/workflows/ci.yml

```diff
@@ -47,11 +47,13 @@ jobs:
           REQUIRE_DEV: true
 
   phpunit:
-    name: "PHPUnit on ${{ matrix.php }} / Castor from ${{ matrix.castor.method }}"
-    runs-on: ubuntu-latest
+    name: "PHPUnit on ${{ matrix.os }} - ${{ matrix.php }} / Castor from ${{ matrix.castor.method }}"
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [ ubuntu-latest, macos-latest ]
+        php: [ "8.1", "8.2", "8.3" ]
```

Comment:
That's too much consumption. The matrix is already quite big. Let's test on linux 8.1 ... 8.3, and only 8.1 on mac